src/resources/styles-data.ts

```diff
@@ -155,3 +152,10 @@ export const derivedStyles = {
   "material-secondary-background-color": "var(--secondary-background-color)",
   "material-secondary-text-color": "var(--secondary-text-color)",
 } as const;
+
+export const legacyStyles = {
+  "state-icon-color": "var(--paper-item-icon-color)",
```

Comment:
This seems like a breaking change? We will now override `state-icon-color` with `paper-item-icon-color` while before we would do it the other way around? Meaning things that where styled with `state-icon-color`  before this PR, are now styled with `paper-item-icon-color`?

Reply:
Oh yeah, I see. Would there be a way to merge this without breaking changes.

Reply:
state-icon-color got introduced somewhere later than paper-item-icon-color, so I can't do it without a breaking change.